README.md

```diff
@@ -17,23 +17,22 @@ Prior to SDK 3.13, `require` paths were handled in a non-standard way. When requ
        at Object.loader.require (loader.js:69:10)
    ```
 
-# Getting Started (SDK)
+# Getting Started (SDK 3.13 or higher)
 
-Clay will eventually be built into the Pebble SDK. However, while it is still in beta you will need to follow the steps shown below:
+Clay is distributed as a [Pebble package](https://developer.pebble.com/guides/pebble-packages/) so it is super easy to include in your project.
 
-1. Download the **clay.js** distribution file from the [latest release](https://github.com/pebble/clay/releases/latest).
-2. Drop `clay.js` in your project's `src/js` directory. 
-3. Create a JSON file called `config.json` and place it in your `src/js` directory. 
-4. In order for JSON files to work you may need to change a line in your `wscript` from `ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob('src/js/**/*.js'))` to `ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob(['src/js/**/*.js', 'src/js/**/*.json']))`.
-5. Your `app.js` file needs to `require` clay and your config file, then be initialized. Clay will by default automatically handle the 'showConfiguration' and 'webviewclosed' events. Copy and paste the following into the top of your `app.js` file:
+1. Run `pebble package install pebble-clay` to install the package in your project
+2. Create a JSON file called `config.json` and place it in your `src/js` directory.
+3. In order for JSON files to work you may need to change a line in your `wscript` from `ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob('src/js/**/*.js'))` to `ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob(['src/js/**/*.js', 'src/js/**/*.json']))`.
+4. Your `app.js` file needs to `require` clay and your config file, then be initialized. Clay will by default automatically handle the 'showConfiguration' and 'webviewclosed' events. Copy and paste the following into the top of your `app.js` file:
 
   ```javascript
-  var Clay = require('./clay');
+  var Clay = require('pebble-clay');
   var clayConfig = require('./config.json');
   var clay = new Clay(clayConfig);
   ```
-6. Add `configurable` to the `capabilities` array in your `appinfo.json`.
-7. Ensure `enableMultiJS` is set to true in your `appinfo.json`.
+6. Add `configurable` to the `pebble.capabilities` array in your `package.json`.
```

Comment:
This shouldn't be necessary. If capabilities are present in the pebble package's package.json file, they will automatically be merged with the capabilities of the app using the pebble package.